src/main/java/httpserver/httpmessages/HTTPResponseParser.java

```diff
@@ -0,0 +1,49 @@
+package httpserver.httpmessages;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class HTTPResponseParser {
```

Comment:
The intention is good by the name is a little missleading:
A parser takes a string and _parses_ it into some other data-structure.
Maybe its more a `writer`? Or `serializer`?